experiments/ClimaCore/sea_breeze/run.jl

```diff
@@ -117,7 +117,7 @@ system before executing the simulation.
 
 t_start, t_end = (0.0, 1e4)
 Δt_coupled = 0.1
-saveat = t_start:10.0:t_end
+saveat = 10.0
```

Comment:
```suggestion
saveat = collect(t_start:10.0:t_end)
```
then just pass this to the component models and we don't need to change the atmos/land/ocean_rhs.jl files.